src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1787,6 +1857,14 @@ static void ConsoleSettings()
 		s_consolePersistentCommandHistory = false;
 		WritePrivateProfileBool("MacroQuest", "ShowMacroQuestConsole", s_consoleVisibleOnStartup, mq::internal_paths::MQini);
 		WritePrivateProfileBool("Console", "PersistentCommandHistory", s_consolePersistentCommandHistory, mq::internal_paths::MQini);
+
+		s_fontSize = s_defaultFontSize;
+		s_currentFontIndex = s_defaultFontIndex;
```

Comment:
reading a bit further, what is the point of `s_fontSize` at all? it always just acts as a temporary variable and the value is never actually used as-is.

Reply:
example given: here you can just write the ini as `s_defaultFontSize` and call `SetFontSize(s_defaultFontSize)`. You're already setting the index here, so just use that in the combo